binder/src/test/java/io/grpc/binder/internal/ServiceBindingTest.java

```diff
@@ -67,6 +67,7 @@ public void setUp() {
 
     shadowApplication = shadowOf(appContext);
     shadowApplication.setComponentNameAndServiceForBindService(serviceComponent, mockBinder);
+    shadowApplication.setUnbindServiceCallsOnServiceDisconnected(false);
```

Comment:
Do we need a test case for when this is true?

Reply:
No because "true" (the default) means preserve a certain legacy robolectric behavior that doesn't match what any version of Android has ever done. I can add a comment to explain.